services/pingpong/pingpong-uptime.tester.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const Joi = require('joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
```

Comment:
Same general theme for this tester as the one for the status tester. Another benefit of using the standard/consistent Shields approach for setting badge attributes is that we'll control what the message can be, and can accordingly reuse our test validator schemas